tools/PI/DevHome.PI/Views/ClipboardMonitoringWindow.xaml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<winex:WindowEx
+    xmlns:winex="using:WinUIEx"
+    x:Class="DevHome.PI.Views.ClipboardMonitoringWindow"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:DevHome.PI.Views"
+    xmlns:controls="using:DevHome.PI.Controls"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Height="80"
+    MaxHeight="80"
+    MinHeight="80"
+    IsMaximizable="False"
+    TaskBarIcon="Images/pi.ico"
+    Title="Project Ironsides Clipboard Monitor">
+
+    <Grid>
+        <controls:ClipboardMonitorControl x:Name="ClipboardMonitorPanel" Visibility="Visible" />
```

Comment:
Is Visible not the default?

Reply:
It is. I've removed the explicit visibility set.